src/main/java/com/kryptnostic/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -598,14 +606,30 @@ public boolean deleteEntitySet( UUID entitySetId ) {
                 .actionGet();
 
         client.admin().indices()
-                .delete( new DeleteIndexRequest( SECURABLE_OBJECT_INDEX_PREFIX + entitySetId.toString() ) );
+                .delete( new DeleteIndexRequest( SECURABLE_OBJECT_INDEX_PREFIX + entitySetId.toString() + "_*" ) );
```

Comment:
You extracted `getIndexName()` and `getTypeName()` as helper functions, so why not for this as well?